dev-libs/tarantool-c/tarantool-c-9999.ebuild

```diff
@@ -0,0 +1,26 @@
+EAPI=6
+
+
+EGIT_REPO_URI="https://github.com/tarantool/tarantool-c.git"
+
+if [[ ${PV} = 9999* ]]; then
+        GIT_ECLASS="git-r3"
+fi
+
+inherit eutils cmake-utils $GIT_ECLASS
+
+DESCRIPTION="Tarantool-c connector. Very fast framework, originaly developed by (C)Mail.ru"
+HOMEPAGE="http://tarantool.github.io/tarantool-c/index.html"
+LICENSE="BSD-2 clause"
```

Comment:
There is `BSD-2` in `/usr/portage/licenses`, so I guess the format should be `LICENSE="BSD-2"`.

Reply:
Ok, changed .

Reply:
Should be `BSD-2`. Anyway, I'm going to change more things. Will push soon.